### PR TITLE
Add Travis support

### LINKED ISSUE #6: Add Travis support
Add Travis support.

### PR DESCRIPTION
**Pull request by szepeviktor**
See https://docs.travis-ci.com/user/languages/php/

Please consider merging my commits. It is possible in GitHub.

Closes #6

### CODE REVIEW

**Comment by szepeviktor** [comment]
Close&reopen to trigger Travis
https://travis-ci.org/szepeviktor/realpath_turbo/builds

**Comment by szepeviktor** [comment]
@Whissi I've hacked it together. https://travis-ci.org/szepeviktor/realpath_turbo/builds/165052643

Waiting for some tests...

`php --rextinfo turbo_realpath` is not really something.

**Comment by szepeviktor** [comment]
Please add Travis badge to readme.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+sudo: required
```
Comment (szepeviktor) [inline .travis.yml]: OK.
How would you install it?

Comment (rayward) [inline .travis.yml]: Here's what I have - https://github.com/bigcommerce/realpath_turbo/pull/1/commits/c59fe50ac3e9900dd95f6af33affe4421a7c8d44

Comment (szepeviktor) [inline .travis.yml]: OK. You may fork [my PR](https://github.com/szepeviktor/realpath_turbo/tree/patch-1).
I suggest to call the ini `realpath_turbo.ini`

**Comment by Whissi** [comment]
Thanks for your help. But I decided to use https://github.com/m6w6/travis-pecl instead.

And finally we are all green now, https://travis-ci.org/Whissi/realpath_turbo/builds/165604130! Wooohooo! :)

**Pull request closed without merging**